scripts/external/snapd/run_device.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Looks like all the `run_device` scripts could be refactored into a single parameterized one, they are very similar